src/20241004_Multi_IncreaseCbBTCSupplyCaps/IncreaseCbBTCSupplyCaps.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Increase cbBTC Supply Caps"
+author: "ACI"
+discussions: "https://governance.aave.com/t/arfc-increase-cbbtc-supply-caps-on-aave-v3-ethereum-market-and-base/19304"
+---
+
+## Simple Summary
+
+This proposal aims to increase the supply caps for cbBTC on both the Aave V3 Ethereum Market and BASE. The increase in supply caps will allow for greater utilization of cbBTC within the Aave ecosystem, potentially improving liquidity and capital efficiency.
+
+## Motivation
+
+The current supply caps for cbBTC on Aave V3 Ethereum and BASE markets may be limiting the potential growth and utilization of this asset, as we’ve seen demand and current Supply Caps not being enough to allocate it.
+
+By increasing these caps, we can:
+
+1. Enhance liquidity for cbBTC within the Aave protocol.
+2. Provide more opportunities for users to supply and borrow cbBTC.
+3. Potentially attract more users and capital to the Aave ecosystem.
+
+## Specification
+
+Ticker: cbBTC (cbBTC)
+
+Mainnet: 0x5c647cE0Ae10658ec44FA4E11A51c96e94efd1Dd
+
+Base: 0xBdb9300b7CDE636d9cD4AFF00f6F009fFBBc8EE6
```

Comment:
```suggestion
Mainnet: 0xcbB7C0000aB88B473b1f5aFd9ef808440eed33Bf

Base: 0xcbB7C0000aB88B473b1f5aFd9ef808440eed33Bf
```

Reply:
Interesting, why the generator put the aToken instead of the underlying?
Maybe a bug somewhere, I can check if I find the time to